### PR TITLE
#4134 Implement new get known platforms for Server Author View service and admin service 

### PR DESCRIPTION
**Pull request by davidradl**
I have created this as a draft. Some decisions I have made in this fix that I would like you to be aware of and feedback.

1. I have introduced a new rest call API .../platform/configurations, this is defined in the ConfigResource class, so it has easy access to the existing admin config related code (specifically the code that retrieves a config). Prior to this fix ConfigResource only exposed calls like /servers/{serverName}/configuration and the like. 
2. The fix as coded, verifies the user as a platform investigator. There is a case to say that this Rest API should be in the platform service, which is where I have put it client side, but then the existing server retrieval code would need to refactored or a dependancies changed between the admin services and platforms services modules.   
3. I have coded up the Server Author View API so I could test the admin client. The Server author view API gets all the known platforms (those in the view service configuration) back. The objects that are returned to the UI are minimal.  
4. As I was in the Server Author View I changed the deploy API slightly, so it issues the admin call to the platform that is accumulating configurations, which then deploys to the destination platform named in the request.

### CODE REVIEW

File: open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/OMAGServerPlatformConfigurationClient.java [hunk 1]
```diff
@@ -252,4 +256,25 @@ public Connection getPlatformSecurityConnection() throws OMAGNotAuthorizedExcept
...
+
+    /**
+     * Return all the the OMAG Server configurations that are stored on this platform
+     *
+     * @return the OMAG Server configurations that are stored on this platform
+     * @throws OMAGNotAuthorizedException the supplied userId is not authorized to issue this command.
+     * @throws OMAGInvalidParameterException invalid parameter.
+     * @throws OMAGConfigurationErrorException unusual state in the admin server.
+     */
+    public Set<OMAGServerConfig> getPlatformServerConfigs() throws OMAGNotAuthorizedException,
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/OMAGServerPlatformConfigurationClient.java:268]: This is a confusing name since the server configurations are not stored in the platform.  They are managed by the config store connector.  This may be being shared by multiple platforms (for example when we run multiple servers in IntelliJ, all of the configuration is visible to all platforms).  A better name would be `getAllServerConfigurations()` since it spells out configuration to match the other methods in this module and does not imply that these configurations are tied to the platform.

File: open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/OMAGServerPlatformConfigurationClient.java [hunk 2]
```diff
@@ -252,4 +256,25 @@ public Connection getPlatformSecurityConnection() throws OMAGNotAuthorizedExcept
...
+                                                                   OMAGConfigurationErrorException,
+                                                                   OMAGInvalidParameterException {
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/OMAGServerPlatformConfigurationClient.java:270]: Code formatting

File: open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-encrypted-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/encryptedfile/DocStoreErrorCode.java [hunk 3]
```diff
@@ -78,6 +78,10 @@
             "Unable to delete the encrypted configuration file {0}; exception was {1} with message {2}.",
             "The system was unable to delete the encrypted configuration file.",
             "Review the full stack trace in the logs to troubleshoot further. Then retry the request."),
+    CONFIG_RETRIEVE_ALL_ERROR(400, "ENCRYPTED-DOC-STORE-400-014 ",
+                        "Unable to retrieve the encrypted configuration files; exception was {0} with message {1}.",
```
Comment (mandy-chessell) [inline open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-encrypted-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/encryptedfile/DocStoreErrorCode.java:82]: Would be useful to include the name of the directory/file it was trying to access.

File: open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/file/DocStoreErrorCode.java [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+ *     </ul></li>
+ *     <li>Error Message Id - to uniquely identify the message</li>
+ *     <li>Error Message Text - includes placeholder to allow additional values to be captured</li>
+ *     <li>SystemAction - describes the result of the error</li>
+ *     <li>UserAction - describes how a user should correct the error</li>
+ * </ul>
+ */
+public enum DocStoreErrorCode implements ExceptionMessageSet
+{
+    CONFIG_RETRIEVE_ALL_ERROR(400, "UNENCRYPTED-DOC-STORE-400-001 ",
```
Comment (mandy-chessell) [inline open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/file/DocStoreErrorCode.java:29]: Again add the directory / file name that is not accessible.  Also a message id prefix such as "FILE-CONFIG-DOC-STORE"
would be much more helpful to identify that is this a configuration document store that uses files - not any old document store.

File: open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/file/FileBasedServerConfigStoreConnector.java [hunk 5]
```diff
@@ -144,7 +153,27 @@ public void removeServerConfig()
 
         configStoreFile.delete();
     }
-
+    @Override
+    public Set<OMAGServerConfig> retrieveAllServerConfigs() {
```
Comment (mandy-chessell) [inline open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/file/FileBasedServerConfigStoreConnector.java:157]: Code formatting

Can not just test for files that end in config - need to use the file formatting string.

File: open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/ffdc/OMAGAdminErrorCode.java [hunk 6]
```diff
@@ -299,6 +299,15 @@
                           "The view service has not been passed valid configuration.",
                           "Check whether the view service expects SolutionViewServiceConfiguration or IntegrationViewServiceConfiguration, correct the configuration and restart the server."),
 
+    RETRIEVE_ALL_CONFIGS_NOT_SUPPORTED(400, "OMAG-ADMIN-400-043",
+                        "A retrieve all configurations has been attempted, but operation is not supported.",
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/ffdc/OMAGAdminErrorCode.java:303]: ... what is not supporting this command?

File: open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/ffdc/OMAGAdminErrorCode.java [hunk 7]
```diff
@@ -299,6 +299,15 @@
...
+                        "The retrieve all server configurations operation is rejected.",
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/ffdc/OMAGAdminErrorCode.java:304]: What has rejected this operation?

File: open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/ffdc/OMAGAdminErrorCode.java [hunk 8]
```diff
@@ -299,6 +299,15 @@
...
+                        "Check whether OMAG Server configuration connector supports retrieve all configurations."),
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/ffdc/OMAGAdminErrorCode.java:305]: What do you do when you have checked?

File: open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/ffdc/OMAGAdminErrorCode.java [hunk 9]
```diff
@@ -299,6 +299,15 @@
...
+    UNABLE_TO_OBTAIN_SERVER_CONFIG_STORE(400, "OMAG-ADMIN-400-044",
+                                         "User {0} has attempted to obtain a server config store to be able to retrieve the OMAG server atored configurations but an error occurred.",
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/ffdc/OMAGAdminErrorCode.java:307]: Typo in `atored`.
What is the error?

File: open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/ffdc/OMAGAdminErrorCode.java [hunk 10]
```diff
@@ -299,6 +299,15 @@
...
+                                         "The retrieve all server configurations operation is rejected, as the OMAG Server Configuration store could not be obtained.",
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/ffdc/OMAGAdminErrorCode.java:308]: Is it the store or is it the connector that can not be obtained?

File: open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/rest/OMAGServerConfigsResponse.java [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+ * OMAGServerConfig objects as a response.
+ */
+@JsonAutoDetect(getterVisibility=PUBLIC_ONLY, setterVisibility=PUBLIC_ONLY, fieldVisibility=NONE)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown=true)
+public class OMAGServerConfigsResponse extends AdminServicesAPIResponse
+{
+    private static final long    serialVersionUID = 1L;
+
+    private Set<OMAGServerConfig> serverConfigs = null;
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/rest/OMAGServerConfigsResponse.java:29]: All other methods in this module use Lists rather than Sets.   Lists have an advantage in that they have an order and that creates a consistency in operation.

File: open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/rest/OMAGServerConfigsResponse.java [hunk 12]
```diff
@@ -0,0 +1,130 @@
...
+
+
+    /**
+     * Return the OMAGServerConfig objects.
+     *
+     * @return OMAGServerConfig objects
+     */
+    public Set<OMAGServerConfig> getOMAGServerConfigs()
+    {
+        if (serverConfigs == null)
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/rest/OMAGServerConfigsResponse.java:47]: Need to test for empty set and return null.  The current logic is just returning the server configs irrespective of value

File: open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/store/OMAGServerConfigStoreRetrieveAll.java [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+package org.odpi.openmetadata.adminservices.store;
+
+import org.odpi.openmetadata.adminservices.configuration.properties.OMAGServerConfig;
+import org.odpi.openmetadata.adminservices.ffdc.exception.OMAGInvalidParameterException;
+import org.odpi.openmetadata.adminservices.ffdc.exception.OMAGNotAuthorizedException;
+
+import java.util.Set;
+
+/**
+ * OMAGServerConfigStoreQueryAll provides a method to retrieve all the stored servers. It is an extension to the OMAGServerConfigStore interface
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/store/OMAGServerConfigStoreRetrieveAll.java:12]: This name does not match the name of the interface.

File: open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerAdminServices.java [hunk 14]
```diff
@@ -3088,6 +3091,36 @@ public OMAGServerConfigResponse getStoredConfiguration(String userId,
...
     }
+    /**
+     * Return the stored configuration documents stored on the platform
+     *
+     * @param userId  user that is issuing the request
+     * @return OMAGServerConfigs properties or
+     * OMAGNotAuthorizedException the supplied userId is not authorized to issue this command or
+     * OMAGInvalidParameterException invalid parameter occurred while processing.
+     */
+    public OMAGServerConfigsResponse retrieveAllServerConfigs(String userId) {
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerAdminServices.java:3102]: Formatting

File: open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerAdminStoreServices.java [hunk 15]
```diff
@@ -215,6 +242,58 @@ private OMAGServerConfigStore getServerConfigStore(String   serverName,
...
+    {
+        Connection   connection = this.getConnectionForRetrieveAll();
+
+        try
+        {
+            ConnectorBroker connectorBroker = new ConnectorBroker();
+
+            Connector connector = connectorBroker.getConnector(connection);
+
+            OMAGServerConfigStore serverConfigStore = (OMAGServerConfigStore) connector;
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerAdminStoreServices.java:263]: Seems that the retrieve all connector should be a different type of connector to the existing configuration store

File: open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerAdminStoreServices.java [hunk 16]
```diff
@@ -215,6 +242,58 @@ private OMAGServerConfigStore getServerConfigStore(String   serverName,
...
+                                                                                                            error.getClass().getName(),
+                                                                                                            error.getMessage()),
+                                                    this.getClass().getName(),
+                                                    methodName,
+                                                    error);
+        }
+    }
+
+    /**
+     * Get hte OMAG Server Config store for retrieving all the server configurations associated with this platform
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerAdminStoreServices.java:280]: Typo

File: open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerAdminStoreServices.java [hunk 17]
```diff
@@ -215,6 +242,58 @@ private OMAGServerConfigStore getServerConfigStore(String   serverName,
...
+     * @param serverConfigStore the server config store- note this may not support this operations
+     * @param methodName current operation
+     * @return the store to use for the retrieve all server configurations
+     * @throws OMAGConfigurationErrorException the store does not support the retrieve all configured servers for this platform operation
+     */
+    OMAGServerConfigStoreRetrieveAll getOMAGServerConfigStoreRetrieveAll(OMAGServerConfigStore serverConfigStore, String  methodName) throws OMAGConfigurationErrorException {
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerAdminStoreServices.java:286]: Code formatting

File: open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerAdminStoreServices.java [hunk 18]
```diff
@@ -344,6 +423,22 @@ void saveServerConfig(String            serverName,
...
+    /**
+     * Retrieve all the saved OMAG Server configurations for this platform.
+     *
+     * @param userId calling user
+     * @param methodName  method requesting the servers details
+     * @return  a set of OMAG Server configurations
+     * @throws OMAGConfigurationErrorException the OMAG Server configuration connector defined in configuration does not support retrieve all servers call.
+     */
+    Set<OMAGServerConfig> retrieveAllServerConfigs(String   userId,
+                                              String   methodName) throws OMAGConfigurationErrorException, OMAGInvalidParameterException {
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerAdminStoreServices.java:435]: Code formatting

File: open-metadata-implementation/admin-services/admin-services-server/src/test/java/org/odpi/openmetadata/adminservices/TestOMAGServerAdminStoreServices.java [hunk 19]
```diff
@@ -0,0 +1,84 @@
...
+import org.odpi.openmetadata.adminservices.configuration.properties.OMAGServerConfig;
+import org.odpi.openmetadata.adminservices.ffdc.exception.OMAGConfigurationErrorException;
+import org.odpi.openmetadata.adminservices.store.OMAGServerConfigStore;
+import org.odpi.openmetadata.adminservices.store.OMAGServerConfigStoreRetrieveAll;
+import org.testng.annotations.Test;
+import java.util.Set;
+import static org.testng.FileAssert.fail;
+
+public class TestOMAGServerAdminStoreServices {
+    /**
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-server/src/test/java/org/odpi/openmetadata/adminservices/TestOMAGServerAdminStoreServices.java:14]: Code formatting

File: open-metadata-implementation/admin-services/admin-services-spring/src/main/java/org/odpi/openmetadata/adminservices/spring/ConfigResource.java [hunk 20]
```diff
@@ -17,7 +18,7 @@
  * configuration document is not found, a new one is created.
  */
 @RestController
-@RequestMapping("/open-metadata/admin-services/users/{userId}/servers/{serverName}")
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-spring/src/main/java/org/odpi/openmetadata/adminservices/spring/ConfigResource.java:20]: Better to put the new method in a new Resource file rather than having to change all of the existing methods

File: open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/OMAGServerPlatformConfigurationClient.java [hunk 21]
```diff
@@ -269,7 +269,7 @@ public Connection getPlatformSecurityConnection() throws OMAGNotAuthorizedExcept
                                                                    OMAGConfigurationErrorException,
                                                                    OMAGInvalidParameterException {
         final String methodName  = "getPlatformServerConfigs";
-        final String urlTemplate = "/open-metadata/admin-services/users/{0}/configurations";
+        final String urlTemplate = "/open-metadata/admin-services/users/{0}/platform/server-configurations";
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/OMAGServerPlatformConfigurationClient.java:272]: should not have platform in the name as the server configuration documents are not associated with a platform

File: open-metadata-implementation/admin-services/admin-services-spring/src/main/java/org/odpi/openmetadata/adminservices/spring/ConfigResource.java [hunk 22]
```diff
@@ -46,16 +46,20 @@ public OMAGServerConfigResponse getStoredConfiguration(@PathVariable String user
         return adminAPI.getStoredConfiguration(userId, serverName);
     }
     /**
-     * Return the configuration documents stored on the platform
+     * Return the server configuration documents stored on the platform
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-spring/src/main/java/org/odpi/openmetadata/adminservices/spring/ConfigResource.java:49]: Not stored in the platform.

File: open-metadata-implementation/admin-services/admin-services-spring/src/main/java/org/odpi/openmetadata/adminservices/spring/ConfigResource.java [hunk 23]
```diff
@@ -46,16 +46,20 @@ public OMAGServerConfigResponse getStoredConfiguration(@PathVariable String user
...
      *
      * @param userId  user that is issuing the request
      * @return OMAGServerConfigs properties or
      * OMAGNotAuthorizedException the supplied userId is not authorized to issue this command or
      * OMAGInvalidParameterException invalid parameter occurred while processing.
      */
-    @GetMapping(path = "/configurations")
+    @GetMapping(path = "/platform/server-configurations")
```
Comment (mandy-chessell) [inline open-metadata-implementation/admin-services/admin-services-spring/src/main/java/org/odpi/openmetadata/adminservices/spring/ConfigResource.java:56]: should be somethin like `.../configurations/all`

**Review by mandy-chessell — CHANGES_REQUESTED** [review]

**Comment by davidradl** [comment]
> Where is this description coming from if the details of the stored server is coming from the configuration documents? Is this a placeholder for when the platform details are retrieved from the metadata server?

Yes it is placeholder.

File: open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/OMAGServerPlatformConfigurationClient.java (hunk 1 above)
Comment (davidradl) [reply open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/OMAGServerPlatformConfigurationClient.java:268]: OK Changing

File: open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/OMAGServerPlatformConfigurationClient.java (hunk 2 above)
Comment (davidradl) [reply open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/OMAGServerPlatformConfigurationClient.java:270]: changing

File: open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/file/FileBasedServerConfigStoreConnector.java (hunk 5 above)
Comment (davidradl) [reply open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/file/FileBasedServerConfigStoreConnector.java:157]: Good spot -I was not aware of that.

File: open-metadata-implementation/admin-services/admin-services-spring/src/main/java/org/odpi/openmetadata/adminservices/spring/ConfigResource.java (hunk 23 above)
Comment (davidradl) [reply open-metadata-implementation/admin-services/admin-services-spring/src/main/java/org/odpi/openmetadata/adminservices/spring/ConfigResource.java:56]: I changed to /configurations , I think this implies that all the configurations will be returned, we we need to filter , page, etc in the future I suggest we add query params for that.

File: open-metadata-implementation/admin-services/admin-services-spring/src/main/java/org/odpi/openmetadata/adminservices/spring/ConfigResource.java (hunk 20 above)
Comment (davidradl) [reply open-metadata-implementation/admin-services/admin-services-spring/src/main/java/org/odpi/openmetadata/adminservices/spring/ConfigResource.java]: changed

File: open-metadata-implementation/admin-services/admin-services-server/src/test/java/org/odpi/openmetadata/adminservices/TestOMAGServerAdminStoreServices.java (hunk 19 above)
Comment (davidradl) [reply open-metadata-implementation/admin-services/admin-services-server/src/test/java/org/odpi/openmetadata/adminservices/TestOMAGServerAdminStoreServices.java:14]: changed

File: open-metadata-implementation/admin-services/admin-services-spring/src/main/java/org/odpi/openmetadata/adminservices/spring/ConfigResource.java (hunk 22 above)
Comment (davidradl) [reply open-metadata-implementation/admin-services/admin-services-spring/src/main/java/org/odpi/openmetadata/adminservices/spring/ConfigResource.java]: removed this comment

File: open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/OMAGServerPlatformConfigurationClient.java (hunk 21 above)
Comment (davidradl) [reply open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/OMAGServerPlatformConfigurationClient.java]: removed the platform

File: open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/file/FileBasedServerConfigStoreConnector.java (hunk 5 above)
Comment (davidradl) [reply open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/file/FileBasedServerConfigStoreConnector.java:166]: changed the code to use the formatting.

File: open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/file/DocStoreErrorCode.java (hunk 4 above)
Comment (davidradl) [reply open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/file/DocStoreErrorCode.java]: done

File: open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-encrypted-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/encryptedfile/DocStoreErrorCode.java (hunk 3 above)
Comment (davidradl) [reply open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-encrypted-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/encryptedfile/DocStoreErrorCode.java]: added

**Comment by davidradl** [comment]
I have made the changes from Mandy's review. I hope to merge this at the end of Tuesday unless I get any more feedback. @cmgrote @grahamwallis @mandy-chessell
